scripts/pricefeeds.py

```diff
@@ -203,8 +203,10 @@ def fetch_from_wallet(rpc):
 
   ## feeds for asset
   result = rpc.get_bitasset_data(asset)
+  asset_data = rpc.get_asset(asset)
   price_median_blockchain[asset] = 0.0
-  price_median_blockchain[asset] = float(result["current_feed"]["settlement_price"]["base"]["amount"]) ### FIXME (is this really 'base')
+  price_median_blockchain[asset] = float(result["current_feed"]["settlement_price"]["base"]["amount"]/result["current_feed"]["settlement_price"]["quote"]["amount"]*(10**(5-asset_data["precision"]))) ### FIXME (is this really 'base')
```

Comment:
Having the core assets precision fixed is not very flexible .. 
Furthermore, if I recall correctly, there is a dictionary `assets` that contains all assets involved and their parameters already .. so no need to do an other get_asset call